package.json

```diff
@@ -45,7 +45,7 @@
   "devDependencies": {
     "@faker-js/faker": "^7.6.0",
     "commander": "^9.4.1",
-    "cypress": "9.5.4",
+    "cypress": "^12.13.0",
```

Comment:
This needs to be fixed to `12.13.0` instead of `^12.13.0`, as our image will only carry one version and will not install newer versions on the fly.

Reply:
This is also specific for the arm64 version to be compiled from source.

Reply:
Thanks.